### PR TITLE
new lifegen patrols, shunned stuff, faith patrol block

### PR DESCRIPTION
**Pull request by jaysoleson**
- new "faith_effects" block for patrols to change a cats faith-- test patrol is still in there
- tweaks to accessory gaining from patrols for o_c1 can get em
- new ceremony text for shunned apprentices and forgiven warriors
- disabled working buttons (queen, mediator, medcat) for shunned cats
- new patrols: shunned defaults for several statuses + 2 and 3 cat dates. some statuses still need shunned default patrols
- also a few more two cat df patrols
- enabled the possibility of shunned-specific DF patrols but shunned cats can still get all the normal ones
- thats all i rememebr :3c

### CODE REVIEW

File: resources/dicts/events/lifegen_events/ceremonies.json [hunk 1]
```diff
@@ -203,6 +203,12 @@
         "The silver glow of the moon casts a gentle light over the Clan as they come together in silent anticipation. With pride evident in their gaze, c_l beckons you forward. Each step you take resonates with purpose and commitment. c_l's tail sweeps around you in a protective embrace, sealing your pledge to always safeguard and nurture the Clan's future. The air is thick with emotions as you are hailed the new permanent queen of c_nClan -- y_c.",
         "In the gentle embrace of dawn, the Clan surrounds you, their eyes reflecting hope and expectation. c_l steps to your side, their presence reassuring. With a shared breath, a bond forged from mutual respect and understanding, you step into your new role. Recognized for your compassion, wisdom, and maternal instincts, the Clan bursts into cheers, celebrating the rise of their new permanent queen y_c."
     ],
+    "queen_ceremony forgiven": [
+        "Despite your rocky history with the Clan, c_l feels confident in granting you your full name. Beneath the Great Rock, they name you y_c, and honor you for your honor1. MENTOR"
```
Comment (sedgestripe) [inline resources/dicts/events/lifegen_events/ceremonies.json]: is this supposed to have MENTOR and NOMENTOR still

File: resources/dicts/lifegen_talk/flirt.json [hunk 2]
```diff
@@ -1903,12 +1903,12 @@
             "you_shunned",
             "they_blind",
             "you_blind",
-            "from_mate",
+            "from_your_mate",
```
Comment (sedgestripe) [inline resources/dicts/lifegen_talk/flirt.json]: "from_mate" is correct

File: scripts/patrol/patrol.py [hunk 3]
```diff
@@ -608,7 +603,7 @@ def decide_if_romantic(romantic_event, patrol_leader, random_cat, patrol_apprent
         print("final romance chance:", chance_of_romance_patrol)
         return not int(random.random() * chance_of_romance_patrol)
 
-    def _filter_patrols(self, possible_patrols: List[PatrolEvent], biome:str, camp:str, current_season:str, patrol_type:str):
+    def _filter_patrols(self, possible_patrols: List[PatrolEvent], biome:str, current_season:str, patrol_type:str):
```
Comment (sedgestripe) [inline scripts/patrol/patrol.py:609]: any reason why you're removing camp?

File: scripts/patrol/patrol.py [hunk 4]
```diff
@@ -26,8 +26,6 @@
 from scripts.patrol.patrol_outcome import PatrolOutcome
 from scripts.cat.cats import Cat
 from scripts.special_dates import get_special_date, contains_special_date_tag
-from scripts.utility import change_clan_relations, change_clan_reputation, get_cluster, get_alive_kits, get_alive_cats, get_alive_apps, get_alive_meds, get_alive_mediators, get_alive_queens, get_alive_elders, get_alive_warriors, get_med_cats, ceremony_text_adjust, \
-    get_current_season, adjust_list_text, ongoing_event_text_adjust, event_text_adjust, create_new_cat, create_outside_cat
```
Comment (sedgestripe) [inline scripts/patrol/patrol.py]: removing these imports breaks some abbreviations

File: scripts/patrol/patrol.py [hunk 5]
```diff
@@ -1250,10 +1247,6 @@ def process_text(self, text, stat_cat:Cat) -> str:
                             'red squirrels', 'gray squirrels', 'rats', ]
         text = text.replace('f_mp_p', str(fst_midprey_plural))
 
-        text2 = self.adjust_txt(text)
-        if text2:
-            text = text2
-
```
Comment (sedgestripe) [inline scripts/patrol/patrol.py]: pls my abbrevs 😭

File: scripts/patrol/patrol_outcome.py [hunk 6]
```diff
@@ -519,6 +525,64 @@ def gather_cat_objects(cat_list, patrol: 'Patrol') -> list:
             
         return " ".join(results)
     
+    def _handle_faith_changes(self, patrol:'Patrol') -> str:
```
Comment (sedgestripe) [inline scripts/patrol/patrol_outcome.py:529]: good stuff

File: scripts/patrol/patrol.py [hunk 7]
```diff
@@ -1457,7 +1451,7 @@ def handle_history(self, cat, condition=None, possible=False, scar=False, death=
                 f"WARNING: No history found for {self.patrol_event.patrol_id}, it may not need one but double check please!")
         if scar and "scar" in self.patrol_event.history_text:
             adjust_text = self.patrol_event.history_text['scar']
-            adjust_text = adjust_text.replace("o_c_n", f"{str(self.other_clan.name)}Clan")
```
Comment (sedgestripe) [inline scripts/patrol/patrol.py:1454]: removing Clan may mess up some clangen patrols

**Review by sedgestripe — CHANGES_REQUESTED** [review]

File: resources/dicts/events/lifegen_events/ceremonies.json (hunk 1 above)
Comment (jaysoleson) [reply resources/dicts/events/lifegen_events/ceremonies.json]: LMAO no it is not . one sec

**Pull request merged**